src/app/components/navigation/nav-entry/nav-child/nav-child.component.html

```diff
@@ -1,19 +1,17 @@
 <ng-container *ngIf="entry.route">
-  <a [routerLink]="entry.route" routerLinkActive="active-link">
-    <mat-list-item>
-      <div matListItemAvatar>
-        <mat-icon class="mat-mdc-list-item-avatar" [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}</mat-icon>
-      </div>
-      <span matListItemTitle> {{ entry.i18n | translate }}</span>
-    </mat-list-item>
+  <a mad-sidebar-item [routerLink]="entry.route" routerLinkActive="active-sidebar-item">
+    <mat-icon *ngIf="sublevel === 0" [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}</mat-icon>
+    <span [ngStyle]="{ 'margin-left': sublevel * 30 + 'px' }"> {{ entry.i18n | translate }}</span>
   </a>
 </ng-container>
 
 <!-- Entries without a route are disabled -->
 <ng-container *ngIf="!entry.route">
   <mat-list-item [disableRipple]="true">
     <div matListItemAvatar>
-      <mat-icon class="mat-mdc-list-item-avatar" [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}</mat-icon>
+      <mat-icon class="mat-mdc-list-item-avatar"
+                [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}
```

Comment:
Maybe this should be inside a function?

Reply:
Fixed. Created a service with calculation style object